modules/oe_whitelabel_starter_event/templates/node--oe-sc-event--teaser.html.twig

```diff
@@ -7,17 +7,19 @@
 {% set _title %}
   <a class="standalone" href="{{ url }}">{{ label }}</a>
 {% endset %}
-{% set _content %}
-  <span class="text-muted text-nowrap me-4-5">{{ content.oe_sc_event_dates }}</span>
-{% endset %}
 {% block content %}
 <article{{attributes}}>
   {{ pattern('card', {
     variant: 'search',
     title: _title,
     text: content.oe_summary,
-    image: image,
-    content: _content
+    image: image ? {
+      path: image.src,
+      alt: image.alt,
+    },
+    meta: [
+      content.oe_sc_event_dates,
```

Comment:
I suspect that `content.oe_sc_event_dates` contains some element properties that should not be treated as render elements. Use `|field_value` to clean up.
```suggestion
      content.oe_sc_event_dates|field_value,
```